src/App/App.js

```diff
@@ -34,23 +36,41 @@ class App extends Component {
 
  showHomePage = async () => {
   try {
-   const movies = await API.getData('movies');
-   this.setState({ movies });
-   this.setState({ pageView: "Home", error: '' });
+    const movies = await API.getData('movies')
+    if(this.state.isLoggedIn === true) {
+      API.getData(`users/${this.state.user.id}/ratings`)
+        .then(ratings => {
+        this.convertRatingsToStarValues(ratings)
+        this.setState({
```

Comment:
App is getting so many states on it. I wonder if this is typical for a real life app?